README.md

```diff
@@ -25,6 +25,21 @@ work within the Twyla platform.
 - `cd` to the directory and run `pipenv install --dev`.
 - Make sure that all tests pass: `pytest`.
 
+## Configuration
+
+The configuration values are read from environment variables whose names should
+have the same prefix. This prefix should be passed to the `EventBus` class on
```

Comment:
Could we possibly add an example of a prefix here? I know it's in the tests  but I think it would be nice to have it immediately in the README as well.

Reply:
Yep.

Reply:
Actually there is an example right after this sentence. Is it not clear enough?

Reply:
Ah wait, you're right -- I got confused with the `split_event_name` method from `queues.py`. Sorry!